src/Module/Tos.php

```diff
@@ -16,6 +16,27 @@
 
 class Tos extends BaseModule
 {
+	// Some text elements we need more then once to keep updating them easy.
+	public $privoperate;
+	public $privdistribute;
+	public $privdelete;
+	public $privblock;
```

Comment:
Please add an additional new line between the class member declaration and the Docblock comment below.